tests/ci/reporter_tests.js

```diff
@@ -768,7 +768,7 @@ describe('test reporters', function() {
       reporter.finish();
       var output = stream.read().toString();
       assert.match(output, /<testsuite name="Testem Tests" tests="1" skipped="0" failures="0" timestamp="(.+)" time="(\d+(\.\d+)?)">/);
-      assert.match(output, /<testcase classname="phantomjs" name="it does &lt;cool> &quot;cool&quot; 'cool' stuff"/);
+      assert.match(output, /<testcase classname="phantomjs" name="it does <cool> &quot;cool&quot; 'cool' stuff" time="0"\/>/);
```

Comment:
Shouldn't this actually be escaped? If I understand the test correctly having a `<` here would mean invalid XML.

Reply:
no idea actually, I have just fixed tests 🙂

Reply:
It looks like this is actually a regression in 0.5.0 and we need to wait for 0.6.0 https://github.com/xmldom/xmldom/pull/199

Reply:
Those changes should be reverted now :-)

Reply:
done, do I need to merge commits into one?

Reply:
That would be appreciated, otherwise 👍

Reply:
done!